trino/dbapi.py

```diff
@@ -329,7 +326,7 @@ def _format_prepared_param(self, param):
         if isinstance(param, dict):
             keys = list(param.keys())
             values = [param[key] for key in keys]
-            return "MAP(%s, %s)" % (
+            return "MAP({}, {})".format(
```

Comment:
Just as a note, when we drop support for 3.5 in the subsequent PR as discussed this could be replaced with an f-string instead.